### PR TITLE
Add namespace metadata to all yaml samples

### PR DESCRIPTION
**Pull request by mchmarny**
Add namespace metadata to all yaml snippets in docs (deployment, component, config, etc) per issue #501

### CODE REVIEW

File: howto/components-scopes/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
   name: statestore
-  namespace: production
```
Comment (msfussell) [inline howto/components-scopes/README.md:20]: This one should not be changed. The example is using production namespace

File: howto/components-scopes/README.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ apiVersion: dapr.io/v1alpha1
...
```
Comment (msfussell) [inline howto/components-scopes/README.md:68]: same here

File: reference/specs/bindings/twitter.md [hunk 3]
```diff
@@ -1,23 +1,24 @@
 # Kubernetes Events Binding Spec
```
Comment (msfussell) [inline reference/specs/bindings/twitter.md:1]: Title for this topic is wrong (although not in this PR)

File: reference/specs/bindings/twitter.md [hunk 4]
```diff
@@ -1,23 +1,24 @@
...
 metadata:
-  name: <name>
+  name: <NAME>
+  namespace: <NAMESPACE>
 spec:
   type: bindings.twitter
   metadata:
   - name: consumerKey
-    value: ''
+    value: ""
```
Comment (msfussell) [inline reference/specs/bindings/twitter.md:13]: Other bindings have examples inline. Are there examples to help people understand how to use this binding?

File: howto/components-scopes/README.md (hunk 1 above)
Comment (mchmarny) [reply howto/components-scopes/README.md]: done

File: reference/specs/bindings/twitter.md (hunk 3 above)
Comment (mchmarny) [reply reference/specs/bindings/twitter.md]: done

File: reference/specs/bindings/twitter.md (hunk 4 above)
Comment (mchmarny) [reply reference/specs/bindings/twitter.md]: done

File: howto/components-scopes/README.md (hunk 2 above)
Comment (mchmarny) [reply howto/components-scopes/README.md]: done

**Pull request merged**